front/pages/api/w/[wId]/data_sources/[name]/managed/update.ts

```diff
@@ -104,6 +104,15 @@ async function handler(
         },
       });
 
+      void sendUserOperationMessage({
```

Comment:
Can we log only if the `lastEditedById` changes?

Reply:
The point is to log when the user went through the OAuth flow, not only when the editedby changed.

Reply:
I don't see a problem with users undergoing the OAuth process multiple times. The key concern, in my opinion, is when a different person initiates the OAuth flow from what we've identified. I just want to ensure that those logs will be actionable.